context.go

```diff
@@ -285,24 +305,71 @@ func (c *Context[H]) CreateBlock() Block[H] {
 		}
 
 		c.block.SetTransactions(txx)
```

Comment:
Is it gonna work with the set of transactions changed between Prepare and CommitAck?

Reply:
See the comment to Block.SetTransactions and to the test implementation, this callback won't be used to overwrite the final Block's transactions.